consensus/interface.go

```diff
@@ -177,6 +193,7 @@ type SigningHandler interface {
 	AggregateSigs(bitmap []byte, epoch uint32) ([]byte, error)
 	SetAggregatedSig([]byte) error
 	Verify(msg []byte, bitmap []byte, epoch uint32) error
+	ShallowClone() SigningHandler
```

Comment:
Should be a clone of the existing object, at least for now.
Will try to see if it can be changed when creating a separate component to be injected in the future

Reply:
ok